langwatch/src/components/traces/TracesMapping.tsx

```diff
@@ -47,7 +47,7 @@ const DATASET_INFERRED_MAPPINGS_BY_NAME_TRANSPOSED = Object.entries(
 ).reduce(
   (acc, [key, value]) => {
     if (acc[value]) {
-      acc[value].push(key);
+      acc[value]!.push(key);
```

Comment:
this one is weird, doesn't fail on vscode, but during build typescript thinks it can be undefined even though it was checked just above